app/views/games/show.html.erb

```diff
@@ -12,25 +12,25 @@
 
 		<table id="board">
```

Comment:
Try to think about how to place the pieces on the board dynamically.. This will support moving of the pieces later on..